binding/web/README.md

```diff
@@ -89,23 +91,52 @@ run:
 npx pvbase64 -h
 ```
 
+The same procedure can be used for the [Rhino context](https://github.com/Picovoice/rhino/tree/master/resources/contexts) (`.rhn`) files.
+
 ### Init options
 
-Rhino saves and caches your model file in IndexedDB to be used by Web Assembly. Use a different `customWritePath`
-variable to hold multiple model values and set the `forceWrite` value to true to force re-save the model file.
-Set `processErrorCallback` to handle errors if an error occurs while transcribing.
-If the model file (`.pv`) changes, `version` should be incremented to force the cached model to be updated.
+Rhino saves and caches your model (`.pv`) and context (`.rhn`) files in the IndexedDB to be used by Web Assembly.
+Use a different `customWritePath` variable to hold multiple model values and set the `forceWrite` value to true to force an overwrite of the model file.
+If the model (`.pv`) or context (`.rhn`) files change, `version` should be incremented to force the cached model to be updated. Either `base64` or `publicPath` must be set to instantiate Rhino. If both are set, Rhino will use the `base64` parameter.
 
 ```typescript
-// these are default
-const options = {
+// Context (.rhn)
+const rhinoContext = {
+  publicPath: ${CONTEXT_RELATIVE_PATH},
+  // or
+  base64: ${CONTEXT_BASE64_STRING},
+
+  // Optionals
+  customWritePath: 'custom_context',
+  forceWrite: true,
+  version: 1,
   sensitivity: 0.5,
+}
+
+// Model (.pv)
+const rhinoModel = {
+  publicPath: ${MODEL_RELATIVE_PATH},
+  // or
+  base64: ${MODEL_BASE64_STRING},
+
+  // Optionals
+  customWritePath: 'custom_model',
+  forceWrite: true,
+  version: 1,
+}
+```
+
+Additional engine options are provided via the `options` parameter.
+Set `processErrorCallback` to handle errors if an error occurs while processing audio.
+Use `endpointDurationSec` and `requireEndpoint` to control the engine's endpointing behaviour.
+An endpoint is a chunk of silence at the end of an utterance that marks the end of spoken command.
+
+```typescript
+// Optional. These are the default values
+const options = {
   endpointDurationSec: 1.0,
   requireEndpoint: true,
   processErrorCallback: (error) => {},
-  customWritePath: "rhino_model",
-  forceWrite: false,
-  version: 1,
 }
 ```
 
```

Comment:
while I was doing porcupine, i noticed we can easily combine the main thread and worker thread sections into one and only mention the difference between these two methods

Reply:
Good points, really helps make the README concise